core/logstore/logstore.go

```diff
@@ -49,6 +49,9 @@ type Logstore interface {
 	// GetLog returns info about a log.
 	GetLog(thread.ID, peer.ID) (thread.LogInfo, error)
 
+	// GetOwnLogs returns info about locally managed logs.
+	GetOwnLogs(thread.ID) ([]thread.LogInfo, error)
```

Comment:
New top-level API on logstore. Only used in one place for now, but could be handy. Could alternatively be called something else, but this seems simple enough.

Reply:
`GetDirectLogs`?

Reply:
Yup, I like that, easy update.